spec.md

```diff
@@ -530,6 +536,53 @@ message PluginCapability {
       // returned by NodeGetInfo to ensure that a given volume is
       // accessible from a given node when scheduling workloads.
       ACCESSIBILITY_CONSTRAINTS = 2;
+
+      // EXPANDABLE_ONLINE indicates that the volume can be expanded
+      // when published to a node.
+      // When a Plugin implements this capability it MUST implement
+      // either EXPAND_VOLUME Controller capability or EXPAND_VOLUME
```

Comment:
nits. Let's be consistent, and use "controller capability" throughout (do not capitalize)